db/migrate/data/20160127210624_convert_configurations_to_settings_changes/vmdb.tmpl.yml

```diff
@@ -332,6 +332,8 @@ workers:
         :amqp_heartbeat: 30
       :event_catcher_amazon:
         :poll: 15.seconds
+      :event_catcher_hawkular:
+        :poll: 15.seconds
```

Comment:
So this is just the sleep within each [work loop](https://github.com/durandom/manageiq/blob/318a31cd28c98d097a3e7ebb7288cc770bd1f9fe/app/models/miq_worker/runner.rb#L351) which defaults to `sleep_poll_normal`

now, [do_work](https://github.com/durandom/manageiq/blob/master/app/models/manageiq/providers/base_manager/event_catcher/runner.rb#L146) only processes the events, but does not fetch them. 
@jshaughn could you check again, if you really see, that your `event_monitor_handle.poll` method get invoked every 15.seconds ?